src/scripts/main.js

```diff
@@ -2,7 +2,18 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
добавляй нові рядки в tbody тег
```suggestion
const table = document.querySelector('.dashboard tbody');
```